download_files

```diff
@@ -162,7 +162,7 @@ for i in *.spec PKGBUILD; do
 
     if [ "$CHANGES_GENERATE" == "enable" ]; then
       # Check old tarball if there are any changes files present, take the first one (if any)
-      #changes_filename=$(tar -tf $BN* 2>/dev/null | sed -e "s|[^/]*/||" | grep -iE "changelog|news|changes" | head -n1)
+      #changes_filename=$(tar -tf $BN* 2>/dev/null | sed -e "s|[^/]*/||" | grep -iE "changelog$|news$|changes$" | head -n1)
```

Comment:
> maybe change grep "" to grep '' to avoid accidental shell expansion?

Done in https://github.com/toabctl/obs-service-download_files/commit/08a0c879af1dca9ea3b242eb77de2912a6d891d9